test/postServiceTest.js

```diff
@@ -0,0 +1,26 @@
+const mocha = require('mocha');
+const sinon = require('sinon');
+const assert = require('assert');
+const Post = require('../models/Post');
+const mongoose = require('../mongoose');
+
+const PostService = require('../services/PostService');
+
+postService = new PostService;
```

Comment:
This declares a global variable `postService`, and also is not how you create new instances.

should be `const postService = new PostService();`

Reply:
fixed